### ISSUE TITLE
Syntax Highlighting is incorrect inside style and script blocks

### ISSUE THREAD

**Issue by brycefranzen** [cjoftheweb/laravel-blade-vscode#11]
VSCode v1.33.1 laravel-blade v1.1.2

Syntax highlighting is incorrect in Style and Script Blocks:
![image](https://user-images.githubusercontent.com/6920030/57388148-7d0e3300-716c-11e9-91da-52b6fe8e69f8.png)
![image](https://user-images.githubusercontent.com/6920030/57388282-b646a300-716c-11e9-8f6e-6aa0d43ed748.png)

However, Atom.io handles these cases correctly:
![image](https://user-images.githubusercontent.com/6920030/57388222-9ca55b80-716c-11e9-8cbf-a71e9274e54c.png)
![image](https://user-images.githubusercontent.com/6920030/57388300-c2326500-716c-11e9-8a4c-b199282c2950.png)

**Comment by cjoftheweb** [comment]
Sounds like this extension needs to be converted from Atom again. I can do that.

**Comment by brycefranzen** [comment]
awesome! Do you have an eta on that? Just curious :)

**Comment by 7twin** [comment]
@cjoftheweb the atom alternatives fix a lot of other issues too, is there any resources on how somebody else could port them as it seems this isn't maintained anymore?

**Comment by cjoftheweb** [comment]
Yes, it's very easy, just [convert this CSON file to JSON](https://github.com/jawee/language-blade/blob/master/grammars/blade.cson).

**Comment by 7twin** [comment]
@cjoftheweb oh wow, didn't expect it to be that easy, though I haven't really worked with language plugins for vsCode before, would it just work if I fork laravel-blade and replace the old https://github.com/cjoftheweb/laravel-blade-vscode/blob/master/syntaxes/blade.tmLanguage.json with the new one after cson to json?

**Comment by cjhowe-us** [comment]
Yes, I put a deprecation notice on this repository - I suggest trying the linked extension. If the language file there is out of date, I suggest making a PR with the converted Atom syntax.
